server/views/pages/environment.njk

```diff
@@ -140,6 +141,7 @@
   <script type="module" src="/assets/govukFrontendInit.js"></script>
   <script src="/assets/moj/all.js"></script>
   <script src="https://www.gstatic.com/charts/loader.js"></script>
+  <script src="https://cdn.plot.ly/plotly-2.32.0.min.js" crossorigin="anonymous" charset="utf-8"></script>
```

Comment:
Better way is to add this to the CSP/helmet config 🙂

Reply:
How?